### PR TITLE
Add preliminary HCV support

### PR DESCRIPTION
**Pull request by neganp**
This pull request adds preliminary support for aligning HCV genomes. Specifically, HCV1a sequences from the NS3, NS5a, and NS5b genes can be aligned.

The aligner has been tested thoroughly against single-nucleotide polymorphisms  (see the [validation script](https://gist.github.com/neganp/e17e821a05ac756af8fe1aacdcabbcc1) for details), but has not yet been tested for performance on in-dels. Work to determine the common positions of in-dels in HCV1a is ongoing; the results will be added in a forthcoming PR.

### CODE REVIEW

File: cmd/nucamino/hcv1a.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	hcv1acli.PerformAlignment(ioParams, self.Genes, self.Goroutines,
+		self.Quiet, alignmentParams)
+
+	return nil
+}
+
+func init() {
+	parser.AddCommand(
+		"hcv1a", "Align HCV-1 genotype A sequences",
+		"Align input sequences to HCV-1 genotype B using the FDA recommended reference (Genebank accession no. NC_004102). Suported genes: NS3, NS5A, NS5B.",
```
Comment (philiptzou) [inline cmd/nucamino/hcv1a.go]: Typo: "Suported" => "Supported"

**Comment by philiptzou** [comment]
@neganp You don't have to close the PR you can just submit a simple fix to the branch and it will appear here automatically

**Comment by neganp** [comment]
Ah shoot. I think this is because I deleted the branch on my end. :frowning:  I may have to re-open the PR.

**Comment by neganp** [comment]
re-opened as 6.

**Pull request closed without merging**